tests/ui-cargo/multiple_config_files/warn/src/main.stderr

```diff
@@ -1,2 +1,2 @@
-Using config file `$SRC_DIR/tests/ui-cargo/multiple_config_files/warn/.clippy.toml`
-Warning: `$SRC_DIR/tests/ui-cargo/multiple_config_files/warn/clippy.toml` will be ignored.
+Using config file `$SRC_DIR/.clippy.toml`
+Warning: `$SRC_DIR/clippy.toml` will be ignored.
```

Comment:
This is a "fallout" by setting `CARGO_MANIFEST_DIR`, but I think this **is** the proper way